examples/archive.html

```diff
@@ -1,65 +1,64 @@
 <!DOCTYPE html>
-<html lang="en">
```

Comment:
Ah, can you please take another look at https://github.com/publiclab/Leaflet.DistortableImage/pull/1154/files to see the double/single quotation mark changes that are still needed? Thank you!

Reply:
On it @jywarren